opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -413,12 +413,11 @@ message HistogramDataPoint {
 
   // explicit_bounds specifies buckets with explicitly defined bounds for values.
   //
-  // This defines size(explicit_bounds) + 1 (= N) buckets. The boundaries for
-  // bucket at index i are:
+  // The boundaries for bucket at index i are:
   //
   // (-infinity, explicit_bounds[i]] for i == 0
-  // (explicit_bounds[i-1], explicit_bounds[i]] for 0 < i < N-1
-  // (explicit_bounds[i], +infinity) for i == N-1
+  // (explicit_bounds[i-1], explicit_bounds[i]] for 0 < i < size(explicit_bounds)
+  // (explicit_bounds[i], +infinity) for i == size(explicit_bounds)
```

Comment:
is this not obvious wrong now? you access explicit_bounds[size(explicit_bounds)]?

Reply:
Yes, you are right. The code also accesses `bounds[i-1]` for `i == N`, I missed that when writing up the proposed comment. I think it should be
```suggestion
  // (explicit_bounds[i-1], +infinity) for i == size(explicit_bounds)
```